readme.md

```diff
@@ -203,9 +203,23 @@ Points-free function definitions look just like normal assignments without `func
 
 TODO
 
-## Guarded Functions
+## Guards
+Guards are boolean expressions that prevent evaluation of code unless the guard evaluates to true.
+It can be simulated thanks to the conditional ternary Operator.
```

Comment:
You don't need ternary. And they're not simulated. They're just part of the js language.
e.g. `a && b && c` 
a guards the evaluation of b which guards the evaluation of c.